developer-docs-site/docs/tutorials/your-first-nft-sdk.md

```diff
@@ -0,0 +1,376 @@
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```python
+:!: static/sdks/python/examples/simple-nft.py section_2
+```
+  </TabItem>
+  <TabItem value="rust" label="Rust">
+
+In progress
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+
+In progress
+  </TabItem>
+</Tabs>
+
+### Step 4.3: Creating blockchain accounts
+
+In Aptos, each account must have an on-chain representation in order to support receive tokens, coins, and interacting in other DApps. In Aptos, an account rep
+resents a medium for storing assets, hence it must be explicitly created. This example leverages the Faucet to create and fund Alice's account and to just crea
+te Bob's:
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```python
+:!: static/sdks/python/examples/simple-nft.py section_3
+```
+  </TabItem>
+  <TabItem value="rust" label="Rust">
+
+In progress
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+
+In progress
+  </TabItem>
+</Tabs>
+
+### Step 4.4: Creating a collection
+
+Now begins the process of creating tokens. First, the creator must create a collection to store the tokens in. A collection can contain 0, 1, or many distinct tokens within it. The collection does not restrict the attributes of the tokens, it is only a container.
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+Your application will call `create_collection`:
+```python
+:!: static/sdks/python/examples/simple-nft.py section_4
+```
+
+`create_collection`'s API, which returns a transaction hash:
+```python
+:!: static/sdks/python/aptos_sdk/client.py create_collection
+```
+  </TabItem>
+  <TabItem value="rust" label="Rust">
+
+In progress
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+
+In progress
+  </TabItem>
+</Tabs>
+
+### Step 4.5: Creating a token
+
+To create a token, the creator must specify an associated collection. A token must be associated with a collection and that collection must have remaining token count that can be minted. There are a lot of attributes associated with a token, but the helper API only exposes the minimal amount required to create static content.
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+Your application will call `create_token`:
+```python
+:!: static/sdks/python/examples/simple-nft.py section_5
+```
+
+`create_token`'s API, which returns a transaction hash:
+```python
+:!: static/sdks/python/aptos_sdk/client.py create_token
+```
+  </TabItem>
+  <TabItem value="rust" label="Rust">
+
+In progress
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+
+In progress
+  </TabItem>
+</Tabs>
+
+### Step 4.6: Reading token and collection metadata
+
+Both the collection and token metadata are stored on the creator's account within their `Collections`. This data is stored within tables within Aptos. The SDKs provide convenience wrappers around querying these specific tables:
```

Comment:
`are stored within the creator's account`? within vs in vs on?, especially since you use `within` right after.

`This data is stored within tables within Aptos` -> I think you should change the second within to `on`, because it reads funny.